### PR TITLE
Add configuration support for existing editorconfig option `excluded_…

### PR DESCRIPTION
**Pull request by mavasani**
…symbol_names` to all DFA rules

All the DFA rules now respect the editorconfig option that allows skipping analysis for specified excluded symbols with `excluded_symbol_names` option.

### CODE REVIEW

File: src/Microsoft.NetCore.Analyzers/Core/Security/DoNotInstallRootCert.cs [hunk 1]
```diff
@@ -119,6 +119,17 @@ public override void Initialize(AnalysisContext context)
                     compilationStartAnalysisContext.RegisterOperationBlockStartAction(
                         (OperationBlockStartAnalysisContext operationBlockStartAnalysisContext) =>
                         {
+                            var owningSymbol = operationBlockStartAnalysisContext.OwningSymbol;
+
+                            // TODO: Handle case when exactly one of the below rules is configured to skip analysis.
```
Comment (mavasani) [inline src/Microsoft.NetCore.Analyzers/Core/Security/DoNotInstallRootCert.cs]: @dotpaul @LLLXXXCCC - I have added some TODOs for security dataflow rules that do batched DFA execution at compilation end. The DFA results only contain the location of the original call site, not the containing symbol, so it wasn't trivial for me to address the TODO.

Comment (mavasani) [reply src/Microsoft.NetCore.Analyzers/Core/Security/DoNotInstallRootCert.cs]: I will file a tracking issue for this.

Comment (dotpaul) [reply src/Microsoft.NetCore.Analyzers/Core/Security/DoNotInstallRootCert.cs]: Do you mean that if the user only configured one rule to skip analysis for a given symbol, then the other rule still runs DFA, so we potentially end up with diagnostics for both rules?

Comment (mavasani) [reply src/Microsoft.NetCore.Analyzers/Core/Security/DoNotInstallRootCert.cs]: Yes, that is correct. I haven't handled that case. User would need to skip both diagnostics simultaneously.

File: src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotInstallRootCertTests.cs [hunk 2]
```diff
@@ -405,6 +407,37 @@ public X509Store GetX509Store()
 }");
         }
 
+        [Theory]
```
Comment (dotpaul) [inline src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotInstallRootCertTests.cs]: [Fact] needed?

File: src/Microsoft.NetCore.Analyzers/Core/Security/DoNotInstallRootCert.cs (hunk 1 above)
Comment (mavasani) [reply src/Microsoft.NetCore.Analyzers/Core/Security/DoNotInstallRootCert.cs]: Filed https://github.com/dotnet/roslyn-analyzers/issues/2706

**Comment by mavasani** [comment]
Thanks @dotpaul

**Pull request merged**